### ISSUE TITLE
Problem

### ISSUE THREAD

**Issue by woolfder** [horacio9a/chaturbate-anonymous#4]
1. Error at startup: python: can't open file cb.py: [Errno 2] No such file or directory
2. ![default](https://user-images.githubusercontent.com/35527835/35041550-30d035ee-fb96-11e7-82d8-54fa265ee7b6.png)

**Comment by horacio9a** [comment]
Check your config.cfg who must be in C:/-cba-py with other files like this:

Directory of C:/-cba-py

07.12.2017.  03:55             4.679 cb.py
07.12.2017.  00:30             3.193 cbr.py
07.12.2017.  03:55             4.374 cbsl.py
06.12.2017.  20:20             2.659 cbslr.py
07.12.2017.  03:56             4.180 cbyt.py
06.12.2017.  20:34             2.666 cbytr.py
07.12.2017.  16:15               236 config.cfg
06.12.2017.  20:06             2.791 README.md
06.12.2017.  20:51               135 Requirements.txt

Default config.cfg is:

[folders]
output_folder: C:/Videos/Chaturbate/
[files]
model_list: **C:/Windows/CB_Model.txt**
rtmpdump: C:/Windows/rtmpdump.exe
ffplay: C:/Windows/ffplay.exe
youtube: C:/Windows/youtube-dl.exe
streamlink: C:/Python27/Scripts/streamlink.exe

Default location for cba.bat is:

C:/Windows/cba.bat

**Comment by woolfder** [comment]
Thanks for the reply. I tried not to change config.cfg and left everything as it was, but the problem all the same. All the files are in C:\Windows. 
Also I have no config for this:youtube: C:/Windows/youtube-dl.exe
streamlink: C:/Python27/Scripts/streamlink.exe
![default](https://user-images.githubusercontent.com/35527835/35122083-ae0da2b2-fcae-11e7-8d8c-9d188eb7b3a8.png)

**Comment by horacio9a** [comment]
From your first screenshot I see that you have this path:

C:/Users/-x???/chaturbate/cb.py 

I don't think you're still correct this.
'cb.py' and other files must be in 'C:/-cba-py' 

I repeat, must be like this:

C:/-cba-py/cb.py
C:/-cba-py/cbr.py
C:/-cba-py/cbsl.py
C:/-cba-py/cbslr.py
C:/-cba-py/cbyt.py
C:/-cba-py/cbytr.py
C:/-cba-py/config.cfg

**Comment by woolfder** [comment]
Yes, my mistake, did as you said. Now the recording starts, but immediately ends. Or it writes that the model is offline.
![1](https://user-images.githubusercontent.com/35527835/35150078-f580b578-fd29-11e7-8ee1-6ec0db2be8b9.png)
![3](https://user-images.githubusercontent.com/35527835/35150086-fe0b2e58-fd29-11e7-9807-ca18157a7256.png)

**Comment by horacio9a** [comment]
I assume you don't have 'rtmpdump.exe' at the location specified in the config.cfg

[folders]
output_folder: C:/Videos/Chaturbate/
[files]
model_list: C:/Windows/CB_Model.txt
rtmpdump: **C:/Windows/rtmpdump.exe**
ffplay: C:/Windows/ffplay.exe
youtube: C:/Windows/youtube-dl.exe
streamlink: C:/Python27/Scripts/streamlink.exe

**Comment by woolfder** [comment]
The fact that the file is in the same directory, which is included in the config.

**Comment by horacio9a** [comment]
Check also 'output_folder:' path in 'config.cfg'
Try to record a number of currently active models, because some model may be offline or hidden or private.

**Comment by woolfder** [comment]
[folders]
output_folder: C:/Chaturbate
[files]
model_list: C:/Windows/CB_Model.txt
rtmpdump: C:/Windows/rtmpdump.exe
ffplay: C:/Windows/ffplay.exe
ffmpeg: C:/Windows/ffmpeg.exe
All files are in these directories. 
Also an error in ffmpeg, the function PLAY works correctly, but the REC function does not work and gives an error.
![12](https://user-images.githubusercontent.com/35527835/35181882-91155f46-fddb-11e7-8eb6-49c3bb9bfa18.png)

**Comment by horacio9a** [comment]
I think that you deliberately make mistakes ?

You've seen the default path is:

output_folder: **C:/Videos/Chaturbate/**

correct your path must be:

output_folder: **C:/Chaturbate/**

your output_folder: **C:/Chaturbate** is incorrect !!!

With this incorrect path your file is saved like:

C:/Chaturbatesweetdoll17_CB_20012018-115926.flv

**Comment by woolfder** [comment]
Unfortunately, the problem is not this, after the configuration change the problem has not disappeared.

**Comment by horacio9a** [comment]
I suppose the problem arises because of the appearance of Russian letters somewhere?
Otherwise if it does ok rtmp recording use it because ffmpeg still has freeze and it is well avoided

**Comment by woolfder** [comment]
Russian letters seems nowhere to be found. But now the problem is different, record is very much freeze.
